tests/core/process.c

```diff
@@ -392,15 +394,15 @@ PROCESS_TESTCASE (process_malloc_ranges)
     free (malloc_buf);
 
     ctx.range.base_address = GUM_ADDRESS (stack_buf);
-    ctx.range.size = stack_buf_size;
+    ctx.range.size = TEST_STACK_BUFFER_SIZE;
     ctx.found = FALSE;
     ctx.found_exact = FALSE;
     gum_process_enumerate_malloc_ranges (malloc_range_check_cb, &ctx);
     g_assert (!ctx.found);
     g_assert (!ctx.found_exact);
   }
 }
-
+#undef TEST_STACK_BUFFER_SIZE
```

Comment:
Let's add a blank line after this `#undef` to avoid gluing it to the `#endif`, making it look related to it.